src/curated-corpus/pages/ProspectingPage/ProspectingPage.tsx

```diff
@@ -615,6 +632,19 @@ export const ProspectingPage: React.FC = (): JSX.Element => {
 
           {prospects &&
             prospects.map((prospect) => {
+              if (prospect.approvedCorpusItem) {
```

Comment:
nit pick: can we do this like we have it above

`prospect.approvedCorpusItem && <ExistingProspectCard ...`

Reply:
Pairing - decided to keep this as is